scripts/build_api_docs.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+# abort if we get any error
+set -e
+
+# always make sure we have a clean workspace
+if ! git diff-index --quiet HEAD --; then
+    echo -e "\033[91mgit workspace is dirty and contains changes\033[0"
+    echo -e "\033[91mmake sure you have a clean workspace before running this script\033[0m"
+    exit 1
+fi
+
+# building grafana packages
+echo "bulding grafana packages..."
+yarn packages:build
```

Comment:
Question - do we need to build the packages, isn't ts compile enough?

Reply:
hmm, I will do some testing around this.

Reply:
just thinking aloud, ts compile is quite fast compared to ts compile + rollup

Reply:
Did a quick test but gets compile error on the grafana-ui. So lets have a look at this after this PR is merged to master, sounds ok?

Reply:
Yes! Not a high prio at all. Probably worth leaving an issue for that tho.